### ISSUE TITLE
How to adjust the position of the table?

### ISSUE THREAD

**Issue by mdynksgl** [wwdablu/SimplyPDF#39]
Hello.
I am trying to create pdf file using this API. First of all, thank you for your effort, I would like to point out that it is very useful.
The question I want to ask is that some of the tables I created in the file are smaller and need to be in the middle of the page. I can adjust it by adding a vertical blank line, the command for this has already been added. However, I cannot change its position horizontally, I add the position of the table to an empty cell, but this time, since the edge lines are drawn, a pleasant image does not emerge. Also, line thicknesses are not adjusted with the given command.
Is there any way or method you can suggest in this regard?

**Comment by wwdablu** [comment]
Hi,

Can you please provide a visual presentation of what you want to achieve so
that I can take a look at it.

Thank you

**Comment by mdynksgl** [comment]
![1](https://github.com/wwdablu/SimplyPDF/assets/65883409/d637856a-3ff7-4b64-9caf-a7331f69192d)
Hello, I want the tables I created to be horizontally in the middle, as seen in the picture. I can increase the margins of the page and draw the table by centering it, but since the library does not allow me to change the margins, the available space of the next page decreases and it does not work for me. However, I couldn't find any variables or commands where I could set the position of the tables.
Also, as far as I understand, once we specify the orientation as horizontal or vertical when we first start the pdf file, we cannot change the page orientation again. For these two reasons, I use the method of creating more than one file and combining those files last.

In conclusion, the library you created did its job well and saved me from a great burden. Thank you for your efforts.

**Comment by wwdablu** [comment]
Understood! Yes. The orientation of the page can only be defined at the beginning of the document and it cannot be modified later. As for the table placement, this is an enhancement and need to be looked into. I am looking into it and would be try to release a patch as and when I get some free time to work on it.

Also, thank you for using my library and am really glad it was of help to you.

**Comment by mdynksgl** [comment]
Thank you for your interest and feedback.  I wish you good work.

**Comment by wwdablu** [comment]
The feature has been implemented and is available in the master branch and in the latest version.

**Comment by mdynksgl** [comment]
> The feature has been implemented and is available in the master branch and in the latest version.

Thank you for your interest and work.
